catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/associations/Associated.java

```diff
@@ -109,7 +108,9 @@ public void putAssociations(String id, Collection<Edge> edges)
                 .distinct()
                 .collect(Collectors.toList());
 
-        Map<String, Metacard> metacards = util.getMetacards(ids, getNonrestrictedTagsFilter())
+        Map<String, Metacard> metacards = util.getMetacards(writableSourceIds,
```

Comment:
This one is my fault. I've used the `util` convention for the EndpointUtil class consistently throughout the intrigue backend, so we should probably keep it util unless/until we change all of them.